lib/all-badge-examples.js

```diff
@@ -1832,6 +1832,30 @@ const allBadgeExamples = [
           'imagelayers'
         ]
       },
+      {
+        title: 'MicroBadger Size',
+        previewUri: '/microbadger/image-size/_/httpd.svg',
+        keywords: [
+          'docker',
+          'microbadger'
```

Comment:
`'microbadger'` is not needed since it is in the title. Seems there are some others that should be cleaned up as well. I'd love to set up [Danger](http://danger.systems/js/) to catch things like this!